src/Modules/CalcOffence.lua

```diff
@@ -4280,9 +4280,8 @@ function calcs.offence(env, actor, activeSkill)
 					end
 				end
 				local effectMod = calcLib.mod(skillModList, dotCfg, "AilmentEffect")
-				igniteStacks = 1
 				if not skillData.triggeredOnDeath then
-					igniteStacks = m_min(maxStacks, (output.HitChance / 100) * globalOutput.IgniteDuration / (globalOutput.HitTime or output.Time))
```

Comment:
Should probably add output.HitChance to the calculation of stacks earlier? Will also get reflected on weighted average damage